### ISSUE TITLE
Elemental turrets balance issue

### ISSUE THREAD

**Issue by psznm** [JBurlison/Pandaros.Settlers#102]
So I've been playing insane difficulty looking at elemental turrets, thinking they are next tier and hoping that they would cut my costs in ammo. Thinking they use little mana or no ammo (only mana in repairs). After I've built couple of them I noticed they actually use elemental stones or void making those turrets completely worthless, especially void turret, making me give up on better future in insane difficulty. I believe elemental turrets being worthless is not intended.

**TLDR** - elemental stones (void) as ammo is insanely overpriced.

**Comment by psznm** [comment]
I've been poking around the mod, and I've found the issue has been introduced in commit 4cf251c6 where elementium cost has been increased drastically (because elemental turrets were kinda cheap). I assume this increase has been done without realisation it is used to craft ammo for mentioned turrets as well.

While it produced quite expensive ammo before, after this commit the cost of that ammo became just unreasonable.
